### ISSUE TITLE
How to train a model using CNTK for image classifucation on multiple tags

### ISSUE THREAD

**Issue by akegramener** [Microsoft/CNTK#2458]
How do you train a model using CNTK where each image can have more than one tag?

**Comment by cha-zhang** [comment]
You may use multiple labels per image, and define your loss function with cross-entropy. Some examples of such cases: 
https://github.com/Microsoft/FERPlus

**Comment by cha-zhang** [comment]
Closing this for now. If there're further CNTK usage questions, please post it on Stack Overflow.
